roles/rust-foundation-project-director.md

```diff
@@ -0,0 +1,24 @@
+# Role Description: Rust Foundation Project Director
+
+Please refer to [Rust Foundation Director Roles & Responsibilities](TODO) for components of the role description that apply to all Foundation Board Directors.
+
+## Goals
+
+- Represent the needs and interests of the Rust Project at large within the Rust Foundation board
+- Act as a conduit of information between the Rust Project and the Rust Foundation
+
+## Time Budget
+
+- 2-5 hours per week
+
+## Tasks / Activities / Responsibilities
+
+
+## Qualities of a Council member (required skills)
+
+- Strong understanding of the Rust Project, its teams, processes, and ecosystem.
+- Ability to prioritize others' needs, representing all viewpoints from the Rust Project.
```

Comment:
I really like these [lines from the Council RFC](https://github.com/rust-lang/rfc-leadership-council/blob/63a867ee7a14599e864b4ccba3964a9f086ae400/text/3392-leadership-council.md?plain=1#L219C7-L224). Can we take those and adjust them to this role?

Reply:
should be good now, marking as resolved